migrations/versions/1fa2b55f58bb_.py

```diff
@@ -0,0 +1,33 @@
+"""empty message
```

Comment:
Umm can do, although the migration file always links to a commit which describes it